test/powershell/engine/Help/HelpSystem.Tests.ps1

```diff
@@ -272,5 +272,18 @@ Describe "Get-Help should find pattern help files" -Tags "CI" {
         )
         $command.Invoke() | Should Be $result
     }
+}
+
+  Describe "Get-Help should find pattern alias" -Tags "CI" {
+    
+    It "Get-Help should find alias as command" {
+       (Get-Help where).Name | Should be "Where-Object"
+    }
+
+    It "Get-Help should find alias with ? pattern" {
+       $help = Get-Help wher?
+       $help.Category | Should be "Alias"
+       $help.Synopsis | Should be "Where-Object"
+    }
```

Comment:
How about adding a test for '*' pattern? Maybe `Get-Help spp*`

Reply:
resolved